src/util/malloc.rs

```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "malloc_jemalloc")]
+pub use jemalloc_sys::{calloc, free, malloc_usable_size};
+
+#[cfg(feature = "malloc_mimalloc")]
+pub use mimalloc_sys::{
+    mi_calloc as calloc, mi_free as free, mi_malloc_usable_size as malloc_usable_size,
+};
+
+// Hoard links dynamically; you must define LD_LIBRARY_PATH
```

Comment:
Is there any reason we have to deal with hoard-sys differently? Do the other two sys crates link to their native libraries statically? Can we link hoard statically?

Reply:
Thanks for your feedback. I wasn't able to figure out how to link Hoard statically myself, so I asked Wenyu for help today and now hoard-sys works in the same way as the other malloc implementations.